processor/src/fast/tests/advice_provider.rs

```diff
@@ -228,18 +229,32 @@ struct ConsistencyHost {
     /// snapshots for example if it's used in a loop.
     snapshots: BTreeMap<u32, Vec<ProcessStateSnapshot>>,
     store: MemMastForestStore,
+    source_manager: Arc<dyn SourceManagerSync>,
```

Comment:
Similar comment to the one above: I believe (but I might be wrong) that we use `ConsistencyHost` only for testing, and so, we should be able to use just the `DefaultSourceManager` here.